### PR TITLE
Docfix: Update Cache config pages

### PR DESCRIPTION
**Pull request by mrijke**
* Update Broker-Config.md Cache options table
 * Update Historical-Config.md Cache options table (broker->historical)

### CODE REVIEW

File: docs/content/Broker-Config.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ You can optionally only configure caching to be enabled on the broker by setting
 
 |Property|Possible Values|Description|Default|
 |--------|---------------|-----------|-------|
-|`druid.broker.cache.useCache`|Enable the cache on the broker.|false|
-|`druid.broker.cache.populateCache`|Populate the cache on the broker.|false|
+|`druid.broker.cache.useCache`|Enable the cache on the broker.|true, false|false|
```
Comment (xvrl) [inline docs/content/Broker-Config.md]: I believe the description and values columns are flipped.

**Comment by mrijke** [comment]
Thanks, overlooked that one. Swapped the column names.

File: docs/content/Broker-Config.md [hunk 2]
```diff
@@ -74,10 +74,10 @@ The broker only uses processing configs for nested groupBy queries.
...
 
-|Property|Possible Values|Description|Default|
-|--------|---------------|-----------|-------|
-|`druid.broker.cache.useCache`|Enable the cache on the broker.|false|
-|`druid.broker.cache.populateCache`|Populate the cache on the broker.|false|
+|Property|Description|Possible Values|Default|
+|--------|-----------|---------------|-------|
+|`druid.broker.cache.useCache`|Enable the cache on the broker.|true, false|false|
+|`druid.broker.cache.populateCache`|Populate the cache on the broker.|true, false|false|
 |`druid.cache.type`|`local`, `memcached`|The type of cache to use for queries.|`local`|
```
Comment (xvrl) [inline docs/content/Broker-Config.md]: now it's flipped for the other options.

**Comment by mrijke** [comment]
Sorry, I pulled the push trigger too soon.
I've updated the column naming to be consistent with the other table (Possible Values before Description).

**Pull request merged**